services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/controllers/data/CmAssessmentController.java

```diff
@@ -68,8 +69,8 @@ public ResponseEntity<Page<ConnectionOfTravelAssessment>> findConnectionOfTravel
             @RequestParam(name = "start_time_utc_millis", required = false) Long startTime,
             @RequestParam(name = "end_time_utc_millis", required = false) Long endTime,
             @RequestParam(name = "latest", required = false, defaultValue = "false") boolean latest,
-            @RequestParam(name = "page", required = false, defaultValue = "0") int page,
-            @RequestParam(name = "size", required = false, defaultValue = "10000") int size,
+            @Parameter(description = "Page number (0-based). If not set, no paging will be applied, but the results will still be limited by size.") @RequestParam(name = "page", required = false) Integer page,
+            @RequestParam(name = "size", required = false, defaultValue = "1000") int size,
             @RequestParam(name = "test", required = false, defaultValue = "false") boolean testData) {
```

Comment:
suggestion(blocking): Use the pageable interface instead of rolling our own. The `@PageableDefault` annotation sets up sane defaults while letting users override the options in the request
```suggestion
            @PageableDefault Pageable pageable,
            @RequestParam(name = "test", required = false, defaultValue = "false") boolean testData) {
```